Source/Core/DolphinQt/MenuBar.cpp

```diff
@@ -1313,24 +1313,64 @@ void MenuBar::GenerateSymbolsFromRSOAuto()
 
       if (!found_addr.has_value())
         break;
+
       next = *found_addr + 1;
 
-      // Get the beginning of the string
-      found_addr = accessors->Search(*found_addr, reinterpret_cast<const u8*>(""), 1, false);
-      if (!found_addr.has_value())
-        continue;
+      auto get_string_len_until_non_ascii = [found_addr]() {
```

Comment:
This should probably be renamed since it grabs the maximum length of a module name (and excludes non printable characters not all non ascii ones). A description should be added to emphasis why it's important.

```C++
// Non-null data can precede the module name.
// Get the maximum name length that a module could have.
auto get_max_module_name_len = 
```